libs/liquidity/src/lib/liquidity-table.spec.tsx

```diff
@@ -65,7 +86,28 @@ describe('LiquidityTable', () => {
       'Created',
       'Updated',
     ];
+    // 5002-LIQP-001
     expect(headers).toHaveLength(expectedHeaders.length);
     expect(headerTexts).toEqual(expectedHeaders);
   });
+
+  it('should be able to sort', async () => {
+    await act(async () => {
+      render(
+        <LiquidityTable rowData={multiRowData} stakeToCcyVolume={'0.3'} />
+      );
+    });
+    const headers = await screen.findAllByRole('columnheader');
+    await userEvent.click(headers[7]);
+
+    const rows = screen.getAllByRole('row');
+    // 5002-LIQP-003
+    // TODO update test to correctly sort.
+    expect(rows[2]).toHaveTextContent(
```

Comment:
index comment applies to this too.. sorry I know this is a draft but as you linked it ;)

Reply:
I've updated this test to just check that AgGrid sort is enabled. Rather than checking individual rows. AgGrid will not re-arrange rows in the DOM, instead it will just arrange them visually with styles.